web/src/main/webapp/WEB-INF/jsp/team/match/mform.jsp

```diff
@@ -0,0 +1,79 @@
+<%@ page contentType="text/html;charset=UTF-8" pageEncoding="utf-8" trimDirectiveWhitespaces="true" session="false" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="my" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+
+<my:pagetemplate title="PrepareMatchCreationForm">
+    <jsp:attribute name="body">
+         <div class="container">
+             <div class="row">
+                 <div class="col-md-2"></div>
+                 <div class="col-md-8">
+
+                     <c:url value="${matchRequest.target}" var="matchRequestActionUrl"/>
+                     <form class="form-horizontal match-form" id="match-form" method="post" modelAttribute="matchRequest" action="${matchRequestActionUrl}" autocomplete="off">
```

Comment:
lepší je použít jako akci `${requestScope['javax.servlet.forward.request_uri']}`, protože to je aktuální url stránky a odesílat formulář na tu samou page na které je, je docela pěkná konvence